kitchen-sink/README.md

```diff
@@ -2,13 +2,13 @@ _By [Stefan Verhoeven](https://orcid.org/0000-0002-5821-2060), [Faruk Diblen](ht
 [Jurriaan H. Spaaks](https://orcid.org/0000-0002-7064-4069), [Adam Belloum](https://orcid.org/0000-0001-6306-6937), and
 [Christiaan Meijer](https://orcid.org/0000-0002-5529-5761)._
 
-# Run C++ on the web with web assembly, web workers, React and Vega
+# C++ web app with WebAssembly, Vega, Web Worker and React
 
 In previous blogs in the series we learned
 
-* [how to compile C++ to WebAssembly](TODO)
+* [using C++ in a web app with WebAssembly](TODO)
 * [how to perform computations without blocking the user interface](TODO)
-* [how to make a nice interactive form](TODO)
+* [how to interact with your C++ web app using React forms](TODO)
```

Comment:
```suggestion
* [how to interact with your C++ web app using React forms](../react/README/md)
```

Reply:
there is a typo. updated the link myself.